code/test_planning_top/behavior_agent/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
What does this file do?

Reply:
This file is part of every ROS package, same as the package.xml